scripts/ci/libraries/_kind.sh

```diff
@@ -365,19 +365,23 @@ function apply_kubernetes_resources() {
 function dump_kubernetes_logs() {
     POD=$(kubectl get pods -o go-template --template '{{range .items}}{{.metadata.name}}{{"\n"}}{{end}}' \
         --cluster "${KUBECTL_CLUSTER_NAME}" | grep airflow | head -1)
+    kubectl get all
+    kubectl get all --namespace kube-system
+    echo "------coredns logs-------"
+    DNS_POD=$(kubectl get pods -o go-template -n kube-system --template '{{range .items}}{{.metadata.name}}{{"\n"}}{{end}}' \
+        --cluster "${KUBECTL_CLUSTER_NAME}" | grep dns | head -1)
```

Comment:
You forgot the logs command :dagger: 

Also an easier way for this

```
```suggestion
  kubectl -n kube-system logs -l k8s-app=kube-dns --all-containers --cluster "${KUBECTL_CLUSTER_NAME}"
```

(Even with coredns, the k8s-app label is still kube-dns)